rules/Php83/Rector/ClassConst/AddTypeToConstRector.php

```diff
@@ -220,6 +306,20 @@ private function getParentReflections(string $className): array
             $currentClassReflection !== $classReflection);
     }
 
+    /**
+     * @return ClassReflection[]
+     */
+    private function getChildClassReflections(string $className): array
+    {
+        if (! $this->reflectionProvider->hasClass($className)) {
+            return [];
+        }
+
+        $currentClassReflection = $this->reflectionProvider->getClass($className);
+
+        return $this->familyRelationsAnalyzer->getChildrenOfClassReflection($currentClassReflection);
```

Comment:
the method removed at PR:

- https://github.com/rectorphp/rector-src/pull/5995

so this change no longer valid, 

Please rebase. You may can revert to original `ignore_inheritance` config ;)